### ISSUE TITLE
ExecutionTimer unit test fails intermittently for Windows GitHub Action

### ISSUE THREAD

**Issue by MikeSullivan7** [mantidproject/mantidimaging#1966]
### Summary

The ExecutionTimerTest.test_execute unit test method is failing occasionally for the Windows GitHub Action workflow. Re-running the test eventually causes it to pass. This seems to be affecting Windows only. Upon running a second time, no error was given.
This is a continuation of issue #1417 which has not been fully solved.
 
### Steps To Reproduce

Requires investigation to see the cause of the failure, perhaps the test needs to be more toleration

### Expected Behaviour

Passes consistently

### Current Behaviour

Fails intermittently

### Context

Current main

### Failure Logs

```
self = <mantidimaging.core.utility.test.execution_timer_test.ExecutionTimerTest testMethod=test_execute>

    def test_execute(self):
        t = ExecutionTimer()
        self.assertEqual(t.total_seconds, None)
        self.assertEqual(str(t), 'Elapsed time: unknown seconds')

        with t:
            self.assertEqual(t.total_seconds, None)

            time.sleep(0.1)

>       self.assertGreaterEqual(t.total_seconds, 0.1)
E       AssertionError: 0.09399999999732245 not greater than or equal to 0.1

mantidimaging\core\utility\test\execution_timer_test.py:23: AssertionError
```

**Comment by samtygier-stfc** [comment]
I think it would be best to change the time in the assert to `0.09` as I don't think time has strict guarantees on accuracy.

**Comment by samtygier-stfc** [comment]
Closed in #2045
